.buildkite/scripts/pipelines/pull_request/pipeline.ts

```diff
@@ -315,6 +320,25 @@ const getPipeline = (filename: string, removeSteps = true) => {
       );
     }
 
+    if (
+      GITHUB_PR_LABELS.includes('ci:perf-check:start') ||
+      GITHUB_PR_LABELS.includes('ci:perf-check:dashboard') ||
+      GITHUB_PR_LABELS.includes('ci:perf-check:so-crud')
+    ) {
+      const journeyGroups = GITHUB_PR_LABELS.split(',')
+        .map((label) => label.trim())
+        .filter((label) => label.startsWith('ci:perf-check:'))
+        .map((label) => label.replace('ci:perf-check:', ''))
+        .join(',');
+      console.log(`perf pipeline with groups: ${journeyGroups}`);
```

Comment:
avoid logging here, anything logged throughout this step would be interpreted as yml to be uploaded to buildkite